README.md

```diff
@@ -0,0 +1,734 @@
+import com.ibm.cloud.sdk.core.http.HttpStatus;
+
+public class AdminrestExample {
+
+
+    private AdminrestExample() {
+    }
+// End Code Setup
+```
+
+
+### Authentication
+---
+Use one of the following methods to authenticate:
+
+- To authenticate using Basic Auth:
+  Place these values into the Authorization header of the HTTP request in the form Basic <credentials> 
+  where <credentials> is the username and password joined by a single colon `:` base64 encoded. 
+  For example:
+  ```sh
+  echo -n "token:<APIKEY>" | base64
+  ```
+
+- To authenticate using a bearer token:
+  To obtain your token using the IBM Cloud CLI, first log in to IBM Cloud, then run the following command:
+  ```
+  ibmcloud iam oauth-tokens
+  ```
+  Place this token in the Authorization header of the HTTP request in the form Bearer. Both API key or JWT tokens are supported.
+
+- To authenticate directly using the api_key:
+  Place the key directly as the value of the X-Auth-Token HTTP header.
+
+#### Example
+
+Here's an example of how to create the authenticator using either an API key or a BEARER_TOKEN
+
+```java
+        // Create Authenticator
+        Authenticator authenticator;
+
+        if (apiKey != null && !apiKey.isEmpty()) {
+            // Create an Basic IAM authenticator.
+            authenticator = new BasicAuthenticator("token", apiKey);
+        } else {
+            // Create an IAM Bearer Token authenticator.
+            authenticator = new BearerTokenAuthenticator(bearerToken);
+        }
+
+        // Create Service - Construct the service client.
+        Adminrest service = new Adminrest(serviceName, authenticator);
+        // End Authenticator
+```
+
+
+### Creating a client for the Admin REST API.
+---
+Create a new service object.
+
+```java
+        // Create Service - Construct the service client.
+        Adminrest service = new Adminrest(serviceName, authenticator);
+        // End Authenticator
+```
+
+
+### Creating a Kafka topic
+---
+To create a Kafka topic the admin REST SDK issues a POST request to the /admin/topics path. 
+The body of the request contains a JSON document, for example:
+```json
+{
+    "name": "topicname",
+    "partitions": 1,
+    "configs": {
+        "retentionMs": 86400000,
+        "cleanupPolicy": "delete"
+    }
+}
+```
+
+The only required field is name. The partitions fields defaults to 1 if not set.
+
+Expected HTTP status codes:
+
+- 202: Topic creation request was accepted.
+- 400: Invalid request JSON.
+- 403: Not authorized to create topic.
+- 422: Semantically invalid request.
+
+If the request to create a Kafka topic succeeds then HTTP status code 202 (Accepted) is returned. If the operation fails then a HTTP status code of 422 (Un-processable Entity) is returned, and a JSON object containing additional information about the failure is returned as the body of the response.
+
+
+
+
+#### Example
+
+```java
+    private static void createTopic(Adminrest service, String topicName) {
```

Comment:
We seem to still have the extra tab in these code snippets

Reply:
Well that is because for Java example the methods are inside the constructor and so are tabbed in in the example.
Not much I can do about that.